docs/user/security/securing-kibana.asciidoc

```diff
@@ -5,79 +5,62 @@
 <titleabbrev>Configure security</titleabbrev>
 ++++
 
-{kib} users have to log in when {stack-security-features} are enabled on your
-cluster. You configure roles for your {kib} users to control what data those
-users can access.
+When you start {es} for the first time, {stack-security-features} are enabled on
+your cluster and TLS is configured automatically. The security configuration
+process generates a password for the `elastic` user and an enrollment token for
+{kib}.
+{ref}/configuring-stack-security.html[Start the {stack} with security enabled]
+and then enroll {kib} as part of the configuration process.
 
-Most requests made through {kib} to {es} are authenticated by using the
-credentials of the logged-in user. There are, however, a few internal requests
-that the {kib} server needs to make to the {es} cluster. For this reason, you
-must configure credentials for the {kib} server to use for those requests.
+You can then log in to {kib} as the `elastic` user to create additional roles
+and users. You can also configure <<elasticsearch-mutual-tls,mutual TLS>>
+between {kib} and {es} where requests are authenticated by both applications.
 
-With {security-features} enabled, if you load a {kib} dashboard that accesses
-data in an index that you are not authorized to view, you get an error that
-indicates the index does not exist. The {security-features} do not currently
-provide a way to control which users can load which dashboards.
+NOTE: With {security-features} enabled, if you load a {kib} dashboard that
+accesses data in an index that you are not authorized to view, you get an error
+that indicates the index does not exist. The {security-features} do not
+currently provide a way to control which users can load which dashboards.
 
-To use {kib} with {security-features}:
+[discrete]
+[[security-configure-settings]]
+=== Configure security settings
 
-. {ref}/configuring-security.html[Configure security in {es}].
-
-. Configure {kib} to use the appropriate built-in user.
-+
---
-Update the following settings in the `kibana.yml` configuration
-file:
-
-[source,yaml]
------------------------------------------------
-elasticsearch.username: "kibana_system"
-elasticsearch.password: "kibanapassword"
------------------------------------------------
-
-The {kib} server submits requests as this user to access the cluster monitoring
-APIs and the `.kibana` index. The server does _not_ need access to user indices.
-
-NOTE: The password for the built-in `kibana_system` user is typically set as part of the security configuration process on {es}. For more
-information, see {ref}/built-in-users.html[Built-in users].
---
+You can configure the following security settings to provide additional
+encryption.
```

Comment:
This statement isn't accurate, and I'm not sure we need it here anyways.
Are you concerned that we instruct the users to set the "encryptionKey" below without telling them why?

The short answer is: if they don't set it, we generate an ephemeral encryption key, but whenever they restart Kibana, all of their sessions are invalidated. And, if they set up multiple Kibana instances, they will invalidate each others' sessions. So you should always set an encryption key when setting up Kibana.

Reply:
Thanks for the context 👍 I'll change this to read:

>Set an encryption key so that sessions are not invalidated. You can optionally configure additional security settings and authentication.